twitter/twitter.php

```diff
@@ -557,28 +559,24 @@ function twitter_post_hook(App $a, array &$b)
 		}
 
 		$condition = ['uri' => $b["thr-parent"], 'uid' => $b["uid"]];
-		$orig_post = Item::selectFirst(['author-nick', 'author-network'], $condition);
-		if (!DBA::isResult($orig_post)) {
+		$thr_parent = Item::selectFirst(['uri', 'extid', 'author-nick', 'author-network'], $condition);
```

Comment:
I realized that "author-link" is missing as well ... I hadn't checked that very well.

Reply:
It looks so much like `author-nick`!